perk-store-webapps/src/main/webapp/vue-app/components/PerkStoreApp.vue

```diff
@@ -305,6 +307,9 @@ export default {
     modifiedProducts: [],
   }),
   computed: {
+    walletUri() {
+      return `${eXo.env.portal.context}/${eXo.env.portal.portalName}/wallet`;
+    },
```

Comment:
We don't need to add that in computed, just add in the data
**walletUri: `$ {eXo.env.portal.context} / $ {eXo.env.portal.portalName} / wallet`;**